src/panels/config/info/ha-config-info.ts

```diff
@@ -36,8 +97,8 @@ class HaConfigInfo extends LitElement {
 
   protected render(): TemplateResult {
     const hass = this.hass;
-    const customUiList: Array<{ name: string; url: string; version: string }> =
-      (window as any).CUSTOM_UI_LIST || [];
+    // const customUiList: Array<{ name: string; url: string; version: string }> =
+    //   (window as any).CUSTOM_UI_LIST || [];
```

Comment:
?

Reply:
Yea its in the description above to add this back. on the to do. Making this a draft

Reply:
Missed it hidden under the screenshot 😄